fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/QueryPlan.java

```diff
@@ -273,12 +282,23 @@ private RelationalResultSet executeExplain(@Nonnull ContinuationImpl parsedConti
                     FieldDescription.primitive("VERSION", Types.INTEGER, DatabaseMetaData.columnNoNulls),
                     FieldDescription.primitive("PLAN_HASH_MODE", Types.VARCHAR, DatabaseMetaData.columnNoNulls)
             );
+            StructMetaData plannerMetricsMetadata = new RelationalStructMetaData(
+                    FieldDescription.primitive("TASK_COUNT", Types.BIGINT, DatabaseMetaData.columnNoNulls),
```

Comment:
Do you know if we have coverage of what happens if there isn't a debugger set (e.g., making sure that we don't return a `null` column here)? This looks right to me, but I just wanted to see if it's covered.

Reply:
There are plenty of tests where we don't have a `Debugger` installed (all that's not EXPLAIN) and this is `null`